concourse/tasks/deploy-to-govuk-paas.yml

```diff
@@ -0,0 +1,86 @@
+platform: linux
+image_resource:
+  type: docker-image
```

Comment:
Another use of _docker-image_ here rather than _registry-image_. My instinct says we can and should change them all, but say if there's some subtlety I'm unaware of

Reply:
these have all been updated